docs/testing-operators.md

```diff
@@ -1,9 +1,36 @@
 # Testing Operators
 
+This guide is targeted towards operator authors who are targetting their operators to be available on OpenShift 4.0
+clusters through OperatorHub and want to test that end to end workflow.
+
 ## Testing the full end-to-end flow
 
-todo: add guide
+### Pre-Requisite
+
+You need to have an account with `quay.io`. If you don't have one you can sign up for it at [quay.io](https://quay.io).
+
+### Create your Quay app-registry repository
+
+OperatorHub uses Quay's application repositories for storing the operator bundle. Follow [these instructions](https://docs.quay.io/guides/create-repo.html)
+to create your own application repository. Please note that the name of the repository should match the name of the
+operator in it's package.
```

Comment:
`in it's package` - can we make it bit more explicit? May be an example here?

Reply:
@tkashem added